cropduster/standalone/models.py

```diff
@@ -64,7 +64,7 @@ class Meta:
     def save(self, **kwargs):
         if not self.md5:
             md5_hash = hashlib.md5()
-            with default_open(self.image.path, mode='rb') as f:
+            with default_open(self.image.name, mode='rb') as f:
```

Comment:
I think `default_open` here is a typo that got missed in a recent PR. This should actually be `with default_storage.open(self.image.name...)` (you'll need to import `default_storage` at the top of the module as well)